guides/session.rst

```diff
@@ -183,10 +183,16 @@ Resetting the Session
 ---------------------
 
 The primary aim for Mink is to provide a single consistent web browsing API
-for acceptance tests. But a very important part in testing is isolation.
-
-Mink provides two very useful methods to isolate tests, which can be used
-in your test's ``teardown`` methods:
+for acceptance tests. But a very important part in testing is isolation.  
+Mink tries to automatically "soft" isolate Scenarios from each other without
```

Comment:
this documentation should not talk about scenarios. Scenarios are a Behat concept, not a Mink one. Mink is not tied to Behat